### PR TITLE
PHP: non-namespaced example

### PR DESCRIPTION
**Pull request by webmaster777**
https://github.com/ryanluker/vscode-coverage-gutters/issues/260#issuecomment-618812480

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -270,7 +270,7 @@
     "vscode": "1.1.34"
   },
   "dependencies": {
-    "@cvrg-report/clover-json": "0.3.0",
+    "@cvrg-report/clover-json": "^0.3.2",
```
Comment (ryanluker) [inline package.json:273]: can you leave this pinned to `0.3.2`?

File: test/extension.test.ts [hunk 2]
```diff
@@ -137,6 +137,28 @@ suite("Extension Tests", function() {
         extension.exports.emptyLastCoverage();
     });
 
+    test("Run display coverage on php test file number 3 @integration", async () => {
```
Comment (ryanluker) [inline test/extension.test.ts:140]: excellent thanks a bunch for adding a test to cover this case!

File: example/php/composer.json [hunk 3]
```diff
@@ -20,12 +20,12 @@
   "license": "MIT",
   "type": "library",
   "require": {
-    "php": ">=5.3.0",
-    "monolog/monolog": "1.*"
+    "php": "^7.2"
```
Comment (ryanluker) [inline example/php/composer.json:23]: thanks for updating this and all the other php files! (I have very little to no experience with it so this is super helpful).

File: package.json (hunk 1 above)
Comment (webmaster777) [reply package.json:273]: Sure, though I can't imagine why you want to pin all these dependencies. Bad experience with B/C breaks between minor versions?

Comment (ryanluker) [reply package.json]: haha yep... normally in most software projects I let these float atleast to the minor / patch but freeze the dependencies for builds via the package-lock.json

**Comment by webmaster777** [comment]
> Thanks again! I will try to get the release to the marketplace sometime this week.

Any chance you can create a new patch release? If there's anything I can do to help the release process, let me know!

**Pull request merged**